lib/inspec/formatters/base.rb

```
@@ -182,7 +183,12 @@ def format_expectation_message(example)
 
     def platform(field)
       return nil if @backend.nil?
-      @backend.platform[field]
+      begin
+        @backend.platform[field]
+      rescue Train::Error => e
+        Inspec::Log.error(e.message)
+        nil
```

I think we should document, why it is okay to ignore the error here